src/tool/rtabtransE1.f90

```diff
@@ -5,11 +5,15 @@ program rtransitiontable
 ! Per Jonssson, Malmo University, August 2014
 
 implicit none
-integer, parameter :: ntrans = 100000
+integer, parameter :: ntrans = 1000000
```

Comment:
What's this constant? I guess there is not easy way to get rid of it, by dynamically deciding how many array elements we need?